### PR TITLE
Add bevy_asset

### PR DESCRIPTION
**Pull request by geirsagberg**
Latest `bevy_main` failed because `bevy_asset` seems to have been moved into an explicit feature:

```
error[E0432]: unresolved import `bevy::asset`                                                                                                                                  
 --> C:\Users\Geir\.cargo\git\checkouts\bevy_asset_loader-61e45c7c12c9b28c\2e9f2a5\bevy_asset_loader\src\asset_collection.rs:3:11
  |
3 | use bevy::asset::HandleUntyped;
  |           ^^^^^ could not find `asset` in `bevy`

error[E0432]: unresolved import `bevy::asset`
 --> C:\Users\Geir\.cargo\git\checkouts\bevy_asset_loader-61e45c7c12c9b28c\2e9f2a5\bevy_asset_loader\src\dynamic_asset.rs:5:11
  |
5 | use bevy::asset::{AssetServer, HandleUntyped};
  |           ^^^^^ could not find `asset` in `bevy`
```

With adding the feature as default, the project builds.

### CODE REVIEW

File: bevy_asset_loader/Cargo.toml [hunk 1]
```diff
@@ -13,6 +13,7 @@ categories = ["game-development"]
 readme = "../README.md"
 
 [features]
+default = ["bevy/bevy_asset"]
```
Comment (NiklasEi) [inline bevy_asset_loader/Cargo.toml:16]: Shouldn't we add this feature on the Bevy dependency directly? If we put it to the default features, the crate does not work with `default-features = false`.

Comment (geirsagberg) [reply bevy_asset_loader/Cargo.toml:16]: You're probably right; very much a Rust and Cargo novice here, enjoying your work a lot though 😊 I will update the PR once I figure out how it should be!

Comment (geirsagberg) [reply bevy_asset_loader/Cargo.toml]: There, should be fixed now.

**Review by NiklasEi — APPROVED** [review]
Thanks for the fix 😊

**Comment by NiklasEi** [comment]
CI is red, because `bevy_common_assets` was already updated to Bevy 0.8
I'll take care of it

**Pull request merged**